Makefile

```diff
@@ -260,37 +260,53 @@ DELIM=::
 endif
 
 
+get-namespace-cmd-arg:
+ifdef namespace
+    EXTRA_PARAMS:=$(EXTRA_PARAMS) --namespace=$(namespace)
```

Comment:
you can create another parameter called "NAMESPACE" and just assign `NAMSPACE=$(namespace)`, then you can still add this as parameter to commands - if it doesn't exist nothing will be affected 
in case this parameter doesn't exist it will do nothing

Reply:
so it's the same as it already is, if it doesn't exists nothing will be affected